service/src/test/java/com/ge/predix/controller/test/PolicyManagementControllerIT.java

```diff
@@ -81,13 +84,17 @@ public void setup() throws Exception {
         this.testZone2 = new TestUtils().createTestZone("PolicyMgmtControllerIT2");
         this.zoneService.upsertZone(this.testZone);
         this.zoneService.upsertZone(this.testZone2);
-        MockSecurityContext.mockSecurityContext(this.testZone);
-        MockAcsRequestContext.mockAcsRequestContext(this.testZone);
         this.policySet = this.jsonUtils.deserializeFromFile("controller-test/complete-sample-policy-set.json",
                 PolicySet.class);
         Assert.assertNotNull(this.policySet, "complete-sample-policy-set.json file not found or invalid");
     }
 
+    @BeforeMethod
```

Comment:
No need for beforeMethod() as only one test policyZoneDoesNotExistException() could overwrite security context with null zone and restore when done.